cypress/e2e/trigger_builder.cy.js

```diff
@@ -636,14 +622,7 @@ describe('Trigger builder tests', () => {
         });
         cy.get('table tr').contains('X-Custom-Header');
         cy.get('table tr').contains('Header value');
-        cy.get('#triggerSource')
-          .invoke('val')
-          .should((triggerSource) => {
-            const trigger = JSON.parse(triggerSource);
-            expect(trigger.action.http_static_headers).to.deep.eq({
-              'X-Custom-Header': 'Header value',
-            });
-          });
```

Comment:
This function is supposed to check that the content of the editor matches what was selected in the UI.
Removing this actually removes the meaning of this test entirely